src/database/models/index.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars */
+import pg from 'pg';
+import dotenv from 'dotenv';
+
+const DATABASE_URI = `postgres://${process.env.DEV_DB_USERNAME}:${process.env.DEV_DB_PASSWORD}@${process.env.DEV_DB_HOSTNAME}:${process.env.DEV_DB_PORT}/${process.env.DEV_DB_NAME}`;
+dotenv.config();
```

Comment:
This line of code is supposed to come before line 5, an explanation is that: You can't access `process.env` before calling `dotenv.config()`

Reply:
Ohh yes, I see. thanks